tasks/74_dna-pairing/dna-pairing.js

```diff
@@ -0,0 +1,19 @@
+export function pairElement(str) {
+  const DNAPairs = {
+    A: 'T',
+    T: 'A',
+    G: 'C',
+    C: 'G',
+  };
+  let result = [];
+  for (let i = 0; i < str.length; i++) {
+    let DNAPair = new Array(str[i]);
```

Comment:
you have array composed of 2 items.
first item is always `str[i]`
second item you can get from object without loop